app/api/router.py

```
@@ -144,3 +149,24 @@ async def create_conversation_and_message(
+
+@router.get("/projects/list")
+async def list_projects(
+    db: Session = Depends(get_db),
+    user=Depends(get_api_key_user),
+):
+    return await ProjectController.get_project_list(user, db)
+
+
+@router.get("/list-available-agents")
+async def list_agents(
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)